awx/ui/client/src/notifications/edit/edit.controller.js

```diff
@@ -35,6 +38,16 @@ export default ['Rest', 'Wait',
                 }
             });
 
+            Rest.setUrl(GetBasePath('notification_templates'));
```

Comment:
If I'm reading this correctly I think this might break an auditor's ability to view a notification template.  Ideally I think we'd want to move any permissions checks out to the resolve block of this particular state - see https://github.com/ansible/awx/blob/devel/awx/ui/client/src/notifications/main.js#L57.

Reply:
Note that we may want to change the text on that error as well since this is the edit route.  We also probably want to mark it for translation (I know there are probably numerous alerts like this that aren't marked but gotta start somewhere).

Reply:
Good catch. Cleaning that up...

Reply:
Checked out the latest changes with a system auditor - looking good

Reply:
I synced up with @keithjgrant about moving the default notification messages to a place where anyone with read access can see them. Not sure if that's the issue being discussed in that thread, but wanted to mention I'm working on that.

This is probably more info than you're looking for, but in case you're interested in more details, here's the progress on getting that working in the API (TLDR - looks like getting the defaults shown in the OPTIONS endpoint - outside of the `PUT` section - may involve some hacks).

![image](https://user-images.githubusercontent.com/4440360/61192003-a57f4580-a665-11e9-9ef3-360e0a91d6fe.png)

Reply:
`default` field is explicitly stripped from `GET` section - working on workaround:
https://github.com/jladdjr/awx/blob/67651fca883ba6a9c97e4ac099266f38ddae5d1d/awx/api/metadata.py#L170-L173

Reply:
Posted update on this ^ [here](https://github.com/ansible/awx/pull/4291#issuecomment-511273210). TLDR - found a workaround - and it works - but need to run it by API team since it involves a framework change that is a bit iffy.

Reply:
Update: got everything worked out here. default is now being shown in GET section.